Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
-git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+git_source(:github) { |repo| 'https://github.com/#{repo}.git' }
```

Comment:
Lint/InterpolationCheck: Interpolation in single quoted string detected. Use double quoted strings if you need interpolation.